packages/crypto/tests/wycheproof_secp256k1.rs

```diff
@@ -0,0 +1,394 @@
+#![allow(clippy::single_match)]
```

Comment:
@chipshort FYI: this is a case in which clippy 1.70.0 fails for code that clippy 1.75.0 accepts (adding a comment in a match arm is now enough to not use `if let`). So having the lower version in CI shows the difference and we can it resolve actively with this allow annotation.